content/en/docs/reference/tools/map-crictl-dockercli.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Mapping from dockercli to crictl
+content_type: reference
+---
+
+{{<caution>}}
+This page is deprecated and will be removed in Kubernetes 1.27.
+{{</caution>}}
+
+`crictl` is a command-line interface for {{<glossary_tooltip term_id="cri" text="CRI">}}-compatible container runtimes.
+You can use it to inspect and debug container runtimes and applications on a
+Kubernetes node. `crictl` and its source are hosted in the
+[cri-tools](https://github.com/kubernetes-sigs/cri-tools) repository.
+
+This page provides a reference for mapping common docker cli commands to
+equivalent crictl commands.
+
+## Mapping from docker cli to crictl
```

Comment:
```suggestion
## Mapping from docker CLI to crictl
```
We usually capitalize abbreviations.
It's OK to skip that as this is moving existing content.